xblockutils/studio_editable.py

```diff
@@ -107,6 +106,15 @@ def _make_field_info(self, field_name, field):
             (DateTime, 'datepicker'),
             (JSONField, 'generic'),  # This is last so as a last resort we display a text field w/ the JSON string
         )
+        if self.service_declaration("i18n"):
+            ugettext = self.ugettext
+        else:
+
+            def ugettext_noop(text):
```

Comment:
I understand the impulse to name this `ugettext_noop` since it doesn't do anything, but you never use that name, except to assign it to `ugettext`, so it seems simpler to just define it as that.